portality/static/js/formulaic.js

```diff
@@ -1350,7 +1350,7 @@ var formulaic = {
 
                 if (val) {
                     if (this.container) {
-                        this.container.html('<strong>Full contents: ' + edges.escapeHtml(val) + '</strong>');
+                        this.container.html('<small><strong>Full contents: ' + edges.escapeHtml(val) + '</strong></small>');
```

Comment:
This change also needs to be made a few lines down (L1357) to cover the initialisation of the form.

In fact, I note there is a pre-existing display bug here, L1357 should also include the prefix `Full contents:` so that when the page loads for the first time the full contents display looks the same as it does if you edit a field.

Reply:
@richard-jones I *think* this fixes that? https://github.com/DOAJ/doaj/pull/2116/commits/e27356e7bc70a980177b85b7d69d2a334f73b746 

Just a sidenote on the "Full contents" text, @dommitchell / @jbarnsby  it’s very likely that this will be presented very differently in future iterations of the man. ed / ed / associate ed.’s view of the applications. I guess we would call it "application review" mode simply (as opposed to just the application itself which is what a publisher would see on the public site). 

This feature was not agreed on the original design (and I did not propose it myself nor discuss it with anyone); it was added a bit later since, I’m guessing, the fields didn’t show the full contents of the user input — I don’t find it ideal as it ends up just repeating information in most of the fields and takes up a lot of vertical space. There are definitely neater solutions that will help shorten the application without creating too much noise on the application review page. 

<img width="749" alt="Screenshot 2022-07-12 at 14 41 53" src="https://user-images.githubusercontent.com/6668406/178504130-27c1f997-66cc-4801-bc41-920c3f884abf.png">

Reply:
@sssoz yes, these were added "back" in ("back" in from the old form) because when someone is reviewing stuff, they need to be able to read the whole value. On smaller screens, the inputted value is often truncated by the field length making it hard to read, and sometimes copy and paste, the whole thing, particularly for longer things like publisher or society names, keywords etc.

So, yes. They def take up vertical space and it would be great to find a better solution. Problem to solve: enabling viewers to see entire values on smaller screens and copy and paste them when needed.